### ISSUE TITLE
Add a scroll to top icon / button for longer blog posts

### ISSUE THREAD

**Issue by samplelittlered** [AnguHashBlog/angular#36]
**Describe the solution you'd like**
For long blog posts there should be a scroll to pop button / icon that returns the user to the beginning of the page 
- for this app version google material icons should be used

**Comment by kapilG0** [comment]
.take

**Comment by kapilG0** [comment]
Hello @monacodelisa . I want to work on that issue but needs to confirm regarding requirement please let me know am i correct or not 
In this should i add one icon in the right side of end of page and if user click on that user should needs to go to top of page is it the functionality which needs to implemented ?
![screenshot](https://github.com/AnguHashBlog/angular/assets/59639410/a58fd83c-9005-425b-928f-c1e69f169363)

**Comment by monacodelisa** [comment]
@kapilG0 yes this is very close, you need to check scroll position and then make the scroll button appear after a certain scroll position. This feature has been implemented in the [PeimeNG app version](https://github.com/AnguHashBlog/angular-primeng/pull/57/files)

Implement it in the same way but the only difference here is that there are no `primeicons` since this app version has no UI, what you need to use is Google Material icons, and you can refer to the header buttons - just make a matching one but with 

```
<span class="material-symbols-outlined">
expand_less
</span>
```

which is one of the arrow up icons. If you have any further questions let us know
